scripts/terminalserver/terminalserver.jl

```diff
@@ -215,11 +191,7 @@ function display(d::InlineDisplay, x)
 end
 
 atreplinit(i->Base.Multimedia.pushdisplay(InlineDisplay()))
-@async while true
-    if isdefined(Base, :active_repl)
-        print("\r")
-        change_module("Main", false)
-        break
-    end
-end
 end
+
+@eval using Revise
+Revise.async_steal_repl_backend()
```

Comment:
This shouldn't be here, right?

Reply:
Ah yes, can I suggest we move to include Revise by default (+ config option for whether to use it)? The eval-in-REPL was quite shonky and Revise is superb- seems like a good move